### PR TITLE
Article store messages

### PR DESCRIPTION
**Pull request by seanwiseman**
A small tidy up from messaging changes.

### CODE REVIEW

File: article-store/events/utils.py [hunk 1]
```diff
@@ -80,7 +80,7 @@ def wrapper(*args, **kwargs):
 
 
 @setup_exchanges
-def send_article_message(msg_type: str, run_id: str, message: Optional[str]) -> None:
+def send_article_message(msg_type: str, run_id: str, message: Optional[str] = '') -> None:
```
Comment (thewilkybarkid) [inline article-store/events/utils.py]: `None` rather than an empty string?

File: article-store/events/tests/test_event_utils.py [hunk 2]
```diff
@@ -9,15 +9,14 @@
...
+
     assert base_msg
     assert len(base_msg['eventId'])
     assert base_msg['happenedAt']
     assert base_msg['type'] == 'foo.test'
-    assert base_msg['data'] == {}
-    assert base_msg['aggregate']['service'] == 'article-store'
-    assert base_msg['aggregate']['name'] == 'article-version'
-    assert base_msg['aggregate']['identifier'] == 'someId'
+    assert base_msg['runId'] == run_id
```
Comment (thewilkybarkid) [inline article-store/events/tests/test_event_utils.py:19]: Yeah, haven't been bothering with tests anywhere. #movefast #breakthings

Comment (seanwiseman) [reply article-store/events/tests/test_event_utils.py:19]: I have not been overdoing them, but having some present but failing was worth the fix.

File: article-store/events/utils.py (hunk 1 above)
Comment (seanwiseman) [reply article-store/events/utils.py]: Could argue it both ways, it's not being manipulated in the function so it `None` would still work.

**Review by thewilkybarkid — APPROVED** [review]

**Pull request merged**